File: sources/com/lumiyaviewer/lumiya/ui/minimap/_$Lambda$HQUtmVzLYkemE78mCklVmVxMXms.java
/*
 * Decompiled with CFR 0.152.
 */
package com.lumiyaviewer.lumiya.ui.minimap;

import com.lumiyaviewer.lumiya.react.Subscription;
import com.lumiyaviewer.lumiya.slproto.users.manager.CurrentLocationInfo;
import com.lumiyaviewer.lumiya.ui.minimap.MinimapActivity;

/*
 * Illegal identifiers - consider using --renameillegalidents true
 */
final class _$Lambda$HQUtmVzLYkemE78mCklVmVxMXms
implements Subscription.OnData {
    private final Object -$f0;

    private final /* synthetic */ void $m$0(Object object) {
        ((MinimapActivity)this.-$f0).-com_lumiyaviewer_lumiya_ui_minimap_MinimapActivity-mthref-0((CurrentLocationInfo)object);
    }

    public /* synthetic */ _$Lambda$HQUtmVzLYkemE78mCklVmVxMXms(Object object) {
        this.-$f0 = object;
    }

    public final void onData(Object object) {
        this.$m$0(object);
    }
}

File: Linkpoint/src/main/app/com/linkpoint/ui/minimap/$Lambda$HQUtmVzLYkemE78mCklVmVxMXms.kt
package com.linkpoint.ui.minimap

import com.linkpoint.react.Subscription
import com.linkpoint.slproto.users.manager.CurrentLocationInfo

/* renamed from: com.linkpoint.ui.minimap.-$Lambda$HQUtmVzLYkemE78mCklVmVxMXms  reason: invalid class name */
/* synthetic */ class $Lambda$HQUtmVzLYkemE78mCklVmVxMXms : Subscription.OnData {

    /* renamed from: -$f0  reason: not valid java name */
    private /* synthetic */ Any f452$f0

    private /* synthetic */ Unit $m$0(Any obj) {
        ((this as MinimapActivity).f452$f0).m639com_lumiyaviewer_lumiya_ui_minimap_MinimapActivitymthref0((CurrentLocationInfo) obj)
    }

    /* synthetic */ $Lambda$HQUtmVzLYkemE78mCklVmVxMXms(Any obj) {
        this.f452$f0 = obj
    }

    fun onData(Any obj): Unit {
        $m$0(obj)
    }
}
